video-library-client/src/services/Helpers.js

```diff
@@ -3,11 +3,11 @@
  * @param {{query: string, variables: object}} body
  */
 const fetchClient = (body) => {
-  return fetch(`${process.env.REACT_APP_API_URL}`, {
+  return fetch(`${process.env.REACT_APP_API_UR_NON_NATIVE}`, {
```

Comment:
```suggestion
  return fetch(`${process.env.REACT_APP_API_URL_NON_NATIVE}`, {
```
Update .env.example also

Reply:
Added